packages/visual-editor/src/components/Dropzone/Dropzone.tsx

```diff
@@ -148,41 +79,47 @@ export function Dropzone({
   // Don't trigger the dropzone when it's the root because then the only hit boxes that show up will be root level zones
   // Exception 1: If it comes from the component list (because we want the component list components to work for all zones
   // Exception 2: If it's a child of a root level zone (because we want to be able to re-order root level containers)
-  const isNotDroppable =
-    zoneId === ROOT_ID && draggedSourceId !== 'component-list' && draggingParentIds.length !== 0;
+
+  const isDropzoneEnabled = () => {
+    if (node?.data.blockId === CONTENTFUL_COMPONENTS.columns.id) {
+      return false;
+    }
+
+    if (isDraggingNewComponent) {
+      return isHoveringZone;
+    }
+
+    const draggingParentIds = getZoneParents(draggedSourceId || '');
+
+    if (!draggingParentIds.length) {
+      if (isRootZone) {
+        return true;
+      }
+      return false;
```

Comment:
Since this returns either way, you could revise it to `return isRootZone`

Reply:
Hah duh, this is funny now that I look at it again 🤣